server/main.go

```diff
@@ -1,71 +1,73 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"os"
-	"sync"
 	"text/template"
-	"time"
 
 	"github.com/gorilla/mux"
 	"github.com/subosito/gotenv"
 )
 
-var StoreData sync.Map
-var DataHost PathHost
-
 func main() {
+
+	var store Store //= &MapSyncStore{}
+	ctx := context.Background()
+	store = NewRedisStore("devredis1:6379", ctx)
```

Comment:
No magic strings like `devredis`, use `.env` file and let `docker-compose.yaml` read the values from there
Here use `os.GetEnv("REDIS_HOST")` or viper, your call